test/.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "brightspace/lit-config",
+  "env": {
+    "es6": true,
+    "mocha": true,
+    "node": true
+  },
+  "rules": {
+    "no-invalid-this": 0,
+    "prefer-arrow-callback": 0
```

Comment:
I needed this to work around some issues with `this` in mocha context.

Reply:
Good to know, I have a test config too that I was eventually going to add to the eslint config repo

Reply:
Sounds good - I figured you had something in the works.  ;)

Reply:
Does `"env": { "mocha": true }` work?

Reply:
Haha wait... I see you already have that. Was that always there?! I'm losing it.

Reply:
That's currently being set.  The issue with the arrow functions is that they don't get their own `this`... so setting the mocha timeout at the highest level (via `this.timeout(...)`), or accessing the test name (via `this.test.fullTitle()`) don't work.  I've come to realize that arrow functions have their place, but not for everything.

Reply:
Cool. Worst case we can collect this into a new eslint config if it makes sense.

Reply:
Was this only an issue for setting the timeout (that was moved to the console command)?  Or do we still need it?

Reply:
Ah nvm, you mentioned above we need it for `this.test.fullTitle()`

Reply:
FYI: the [Mocha documentation](https://mochajs.org/#arrow-functions) also discourages use of arrow functions for the reason of not being able to access the Mocha context within these functions.